src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -41,16 +41,19 @@
     LIBFUZZER_CRASH_TYPE_REGEX.format(type='timeout'))
 
 # Regular expressions to detect different sections of logs.
+LIBFUZZER_EXTRA_COUNTERS_REGEX = re.compile(r'^INFO:\s+\d+\s+Extra\s+Counters')
 LIBFUZZER_FUZZING_STRATEGIES = re.compile(r'cf::fuzzing_strategies:\s*(.*)')
 LIBFUZZER_LOG_COVERAGE_REGEX = re.compile(r'#\d+.*cov:\s+(\d+)\s+ft:\s+(\d+).*')
 LIBFUZZER_LOG_DICTIONARY_REGEX = re.compile(r'Dictionary: \d+ entries')
-LIBFUZZER_LOG_END_REGEX = re.compile(r'Done \d+ runs.*')
+LIBFUZZER_LOG_END_REGEX = re.compile(r'Done\s+\d+\s+runs.*')
 LIBFUZZER_LOG_IGNORE_REGEX = re.compile(r'.*WARNING:.*Sanitizer')
-LIBFUZZER_LOG_LINE_REGEX = re.compile(r'^#\d+.*(READ|cov:)')
+LIBFUZZER_LOG_LINE_REGEX = re.compile(r'^#\d+.*(READ|NEW|pulse|REDUCE|ft:)')
```

Comment:
Can ft: exist on its own. Since cov: is removed, why need ft: ?

Reply:
There is a RELOAD as well to add here.

Reply:
Yes, if you compile with e.g. only `-fsanitize-coverage=inline-8bit-counters` you'll get only features without edges.

Added `RELOAD`, but I don't think it ever happened on CF, as it requires either old style `-workers=N` parallelism or any other process that could write into the output corpus directory concurrently with the fuzzer. 

The fork mode uses an isolated dir and disables reload explicitly.

Reply:
Sorry, i meant if "|ft:" is needed in regex since every line has to start with (READ|NEW|pulse|REDUCE|RELOAD) and then either cov: or ft: . either you should add like
(READ|NEW|pulse|REDUCE|RELOAD)\s*(cov|ft): ? or skip "s*(cov|ft):" altogether ?

Reply:
Yeah, can remove it, but have to add `INITED` and `DONE` as well, which makes sense I think.

Reply:
yes right, those two make sense to add, then can remove ft: as it looks weird without cov: